src/core_modules/capture-core/components/ListView/Menu/ListViewMenu.component.tsx

```diff
@@ -89,12 +86,12 @@ const ListViewMenuPlain = ({ customMenuContents = [], classes }: Props) => {
                 <IconMore24 />
             </IconButton>
             {actionsIsOpen && (
-                <Layer onBackdropClick={() => setActionsIsOpen(false)} transparent>
+                <Layer onBackdropClick={() => setActionsIsOpen(false)}>
                     <Popper
                         placement="bottom-end"
-                        reference={anchorRef}
+                        reference={anchorRef.current as any}
```

Comment:
Rollback this change and use `const anchorRef = useRef<any | null>(null);` on line 31